e2e/utils/index.ts

```diff
@@ -754,3 +754,23 @@ export function expectNoAngularDevkit() {
   const result = runCommand(`${list} @angular-devkit/core`);
   expect(result).not.toContain('@angular-devkit/core');
 }
+
+export function waitUntil(
+  predicate: () => boolean,
+  timeout = 1000,
+  ms = 50
+): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const t = setInterval(() => {
+      if (predicate()) {
+        clearInterval(t);
+        resolve();
+      }
+    }, ms);
+
+    setTimeout(() => {
+      clearTimeout(t);
```

Comment:
question: is `t` supposed to be the `setTimeout` return? or `clearTimeout()` supposed to be `clearInterval`? or this is intended?

Reply:
It should be `clearInterval` to stop the interval.